core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/AwsV4AuthScheme.java

```diff
@@ -53,15 +53,11 @@ default String schemeId() {
      * Retrieve the {@link AwsCredentialsIdentity} based {@link IdentityProvider} associated with this authentication scheme.
      */
     @Override
-    default IdentityProvider<AwsCredentialsIdentity> identityProvider(IdentityProviderConfiguration providers) {
-        return providers.identityProvider(AwsCredentialsIdentity.class);
-    }
+    IdentityProvider<AwsCredentialsIdentity> identityProvider(IdentityProviderConfiguration providers);
 
     /**
      * Retrieve the {@link AwsV4HttpSigner} associated with this authentication scheme.
      */
     @Override
-    default AwsV4HttpSigner signer() {
-        return AwsV4HttpSigner.create();
-    }
+    AwsV4HttpSigner signer();
```

Comment:
note (applies to other schemes too), this is redefined in this sub-interface to define the more specific HttpSigner in the return type. And to give a more specific javadoc than the generic one in the parent interface.